src/spectral.ts

```diff
@@ -3,13 +3,15 @@ import { Resolver } from '@stoplight/json-ref-resolver';
 import { DiagnosticSeverity, Dictionary, Optional } from '@stoplight/types';
 import { YamlParserResult } from '@stoplight/yaml';
 import { memoize, merge } from 'lodash';
+import { Agent } from 'http';
```

Comment:
oh, and let's make sure we import only type.
```suggestion
import type { Agent } from 'http';
```